components/webui/package.json

```
@@ -2,21 +2,26 @@
   "name": "webui",
   "version": "0.1.0",
   "description": "",
+  "workspaces": [
+    "common",
+    "client",
+    "server"
+  ],
   "scripts": {
-    "client:lint:check": "cd client && npm run lint:check",
-    "client:lint:fix": "cd client && npm run lint:fix",
-    "client:start": "cd client && npm start",
-    "init": "npm clean-install && (cd client && npm clean-install) && (cd server && npm clean-install)",
-    "lint:check": "npm run client:lint:check && npm run server:lint:check",
-    "lint:fix": "npm run client:lint:fix && npm run server:lint:fix",
-    "server:lint:check": "cd server && npm run lint:check",
-    "server:lint:fix": "cd server && npm run lint:fix",
-    "server:start": "cd server && npm start",
+    "client:lint:check": "npm run lint:check --workspace=client",
+    "client:lint:fix": "npm run lint:fix --workspace=client",
+    "client:start": "npm start --workspace=client",
+    "init": "npm clean-install --workspaces",
+    "lint:check": "npm run lint:check --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
```

so far they shall all be present
```suggestion
    "lint:check": "npm run lint:check --workspaces",
    "lint:fix": "npm run lint:fix --workspaces",
```